k8_cortx_cloud/upgrade-cortx-cloud.sh

```diff
@@ -53,6 +53,11 @@ Usage:
 Options:
     -h              Prints help information.
     -i <IMAGE>      REQUIRED. The name of the container image to upgrade to.
+                    This image may be any of the three CORTX Docker images
```

Comment:
Instead of "Docker images", use "container images" .

Reply:
@osowski , will do.
@keithpine, good idea!  I'll add a separate story for that.